source/fuzz/transformation_wrap_vector_synonym.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2021 Shiyu Liu
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_wrap_vector_synonym.h"
+#include <algorithm>
```

Comment:
Use a line of whitespace before and after the include of <algorithm>. See [names and order of includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) for details.

It's not actually clear to me that you need <algorithm> here but maybe I missed it.

But you do use `unordered_set`, so strictly that should be included (in the spirit of "[include what you use](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use)").

Reply:
Done.